tests/docker_scripts/stateless_runner.sh

```diff
@@ -80,6 +80,9 @@ fi
 
 export IS_FLAKY_CHECK=0
 
+# Export NUM_TRIES so python scripts will see its value as env variable
+export NUM_TRIES
```

Comment:
Here: https://github.com/ClickHouse/ClickHouse/blob/560d6f512890f8e47054432f81a806ee43077086/tests/docker_scripts/process_functional_tests_result.py#L164
We got `NUM_TRIES=1` always and set `is_flaky_check=false`